package/com.unity.formats.usd/Tests/USD.NET/OverrideTests.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Jeremy Cowles. All rights reserved.
+// Copyright 2018 Jeremy Cowles & 2023 Unity Technologies. All rights reserved.
```

Comment:
WE should split this into 2 lines as before.

Reply:
split them up as separate lines for each of the copyright